src/components/pagination/index.md.njk

```diff
@@ -0,0 +1,107 @@
+
+## When to use this component
+
+Use this component when you need to show a list of information, such as search results, across multiple pages.
+
+You might do this when:
+- showing the full list on a single page makes the page too long to load
+- the results further down the list are not needed by most users
+
+## When not to use this component
+
+Only break up lists into pages if it improves the performance or usability of your service.
+
+Do not use this component to link sections together as a multi-page article.
+Instead, use links named after the title of each section. Also, [see the Step by Step navigation pattern](/patterns/step-by-step-navigation/).
+
+Avoid using the 'infinite scroll' technique to automatically load content when the user approaches the bottom of the page. This causes problems for keyboard users.
+
+## How it works
+
+Add the pagination component after the main content of each page in the list of
+results.
+
+[View an example of the component in a standard GOV.UK page template](in-page/index.html).
+
+Show the page number to the title of the page so the user can see it in browser tabs and bookmarks. For example 'Search results (page 1 of 4)'.
+
+Do not show pagination if there's only one page of results.
+
+Redirect users to the first page if they enter a URL with a page number that does not exist.
+
+### Previous and next links
+
+Previous and next links are labelled 'Previous page' and 'Next page'. The word
+'page' is visually hidden to conserve horizontal space.
+
+{{ example({group: "components", item: "pagination", example: "default", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+You can provide custom text for previous and next links if you need to.
+
+{{ example({group: "components", item: "pagination", example: "custom-link-text", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+Do not show the previous page link on the first page — and do not show the next page link on the last page.
+
+{{ example({group: "components", item: "pagination", example: "first-page", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+{{ example({group: "components", item: "pagination", example: "last-page", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+### Showing large numbers of pages
+
+Use ellipses (`...`) to skip over pages when needed.
+
+{{ example({group: "components", item: "pagination", example: "elipses", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+Adjust the page numbers shown to fit the horizontal space available.
+
+Be sure to:
+- show the first, last and current page
+- show at least one page immediately before and after the current page
+- replace gaps of 2 or more pages with elipses (`...`)
+
+For example:
+- **1** 2 … 100
+- 1 **2** 3 … 100
+- 1 2 **3** 4 … 100
+- 1 2 3 **4** 5 … 100
+- 1 … 4 **5** 6 … 100
+- 1 … 97 **98** 99 100
+- 1 … 98 **99** 100
+- 1 … 99 **100**
```

Comment:
Not ideal for readability to show the current page numbers with bold text only (obviously this is fine when coded up with hyperlinks). 

I wonder if there's another way to represent this list?

Reply:
![image](https://user-images.githubusercontent.com/1590604/140508777-646f4d27-8343-46c5-ac60-686a2d6e795b.png)
Something like this?